web/store/index.js

```diff
@@ -144,7 +141,9 @@ export default new Vuex.Store({
         csrf: state.auth.csrf,
         data,
       };
-      return apiFetch(url, options);
+      const studyRequestSaved = apiFetch(url, options);
```

Comment:
while this will still work (the `async` function is still returning a promise), it's a bit misleading: we might expect `studyRequestSaved` to be the study request, not a `Promise` that resolves to the study request.

since it's still a good idea to give quick user-facing feedback, I'd suggest:

```js
commit('setToast', REQUEST_STUDY_SUBMITTED);
return apiFetch(url, options);
```